### ISSUE TITLE
[HOLD App #13216] The avatar of group chats change when hovering report rows.

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#14027]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the app, login with any account that has many group chats.
2. In LHN, try to hover the group chats from the top.
3. Notice that there are few avatars of group chats at the bottom change after that. In case you can't reproduce, try to force refresh and do it over again.

## Expected Result:  
The group chats avatar should be consistent.

## Actual Result :  
There are few avatars of group chats at the bottom change after that.

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?

- [ ] Windows / Chrome /

**Version Number:** 1.2.47-0
**Reproducible in staging?:** Needs reproduction  
**Reproducible in production?:** Needs reproduction
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/210838529-856c8bbb-c8d3-44a0-8e52-8436229fd04f.mov

**Expensify/Expensify Issue URL:**
**Issue reported by:** @hungvu193 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1672417924352099

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~015fe598482c1ea154</li>
        <li>Upwork Job ID: 1611126495504691200</li>
        <li>Last Price Increase: 2023-01-05</li>
    </ul>
</details>

**Comment by luacmartins** [comment]
I can't reproduce this.

**Comment by luacmartins** [comment]
Nvm I was able to reproduce, but needed to do the following:

1. Open App on web
2. Switch to a different tab and then back to the App tab
3. Notice that one of the group chat avatars is now grey and when you hover you it flickers.

**Comment by luacmartins** [comment]
Actually, I'm gonna start working on the Manual Requests doc next week and my plate will be quite full with that. Making this External.

**Comment by luacmartins** [comment]
cc @hungvu193 since you reported the original issue, maybe you wanna work on a solution too?

**Comment by hungvu193** [comment]
@luacmartins yeah I'm happy to do that

**Comment by hungvu193** [comment]
@luacmartins So this issue only happened with the users that didn't have avatar. Since I changed avatar of the user inside group that flickers, this issue didn't happen anymore.
So I think that we already have plan to improve the default avatar, do you think we should put this on hold until this PR (https://github.com/Expensify/App/pull/13216) is merged?

**Comment by luacmartins** [comment]
@hungvu193 sounds good to me. Let's hold on https://github.com/Expensify/App/pull/13216

**Comment by luacmartins** [comment]
I'll make this internal again until https://github.com/Expensify/App/pull/13216 is merged and we can confirm this is still an issue.

**Comment by luacmartins** [comment]
Moving this to a weekly as well!

**Comment by Beamanator** [comment]
FYI Adding to image enhancements tracker, just to keep an eye on these types of things: https://github.com/Expensify/App/issues/10894

**Comment by ali-thowfeek** [comment]
@luacmartins I've found a fix for this. Do you want to explore or hold on until the other PR merges?

**Comment by luacmartins** [comment]
You can always post your solution. If the underlying issue is not resolved once the other PR is merged we will consider your solution, but for now this is on hold.

**Comment by luacmartins** [comment]
That's interesting. If that's the issue, I'm not sure that we would solve it by improving our default avatars and maybe we should implement this solution. That being said, I haven't been able to reproduce this issue in dev anymore. @eVoloshchak can you still reproduce this?

**Comment by eVoloshchak** [comment]
> @eVoloshchak can you still reproduce this?

Can't reproduce it, tried dev, prod, and staging

**Comment by luacmartins** [comment]
We can't reproduce this issue anymore. I'm gonna close it for now. Feel free to reopen if it happens again!
